### ISSUE TITLE
Tab order of footer elements

### ISSUE THREAD

**Issue by cindyli** [inclusive-design/wecount.inclusivedesign.ca#241]
## Describe the bug

When using "tab" key to move through link elements in the footer, the current tab order is: "contact us" section -> "social media" section -> "funders" section.

The correct tab order should be same as the visual order: "social media" section -> "contact us" section -> "funders" section.

## To Reproduce

1. Use "tab" to move the focus until the footer section;
2. Continue to tab, the focus moves through footer links in the wrong order described above.

## Expected behavior

The tab order of footer elements should be in the correct order described above.
